src/components/CommentCard.jsx

```diff
@@ -10,7 +15,14 @@ export const CommentCard = ({
       </p>
       <p className="commentCard__body">{body}</p>
       <p>
-        Votes: <b>{votes}</b>
+        Votes: <b>{votes}</b>{" "}
+        <CommentDeletion
```

Comment:
I wouldn't render the delete button for every comment, just for the comment belong to the logged in user